### PR TITLE
Adjust metrics and dashboard

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- Change all the prometheus opts names from `assetstore` or `cms` to `rafter`

**Related issue(s)**

[Issue #36](https://github.com/kyma-project/rafter/issues/36)

### CODE REVIEW

File: internal/route/extract.go [hunk 1]
```diff
@@ -21,11 +21,11 @@ import (
 
 var (
 	httpServeHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
-		Name: "assetstore_metadata_service_http_request_duration_seconds",
+		Name: "rafter_metadata_service_http_request_duration_seconds",
```
Comment (kazydek) [inline internal/route/extract.go]:
```suggestion
		Name: "rafter_front-matter_service_http_request_duration_seconds",
```

File: internal/route/extract.go [hunk 2]
```diff
@@ -21,11 +21,11 @@ import (
...
 		Help: "Request's duration distribution",
 	})
 	statusCodesCounter = promauto.NewCounterVec(prometheus.CounterOpts{
-		Name: "assetstore_metadata_service_http_request_returned_status_code",
+		Name: "rafter_metadata_service_http_request_returned_status_code",
```
Comment (kazydek) [inline internal/route/extract.go]:
```suggestion
		Name: "rafter_front_matter_service_http_request_returned_status_code",
```

File: pkg/extractor/extractor.go [hunk 3]
```diff
@@ -23,7 +23,7 @@ type extractor struct {
 
 var (
 	readingMetadataHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
-		Name:    "assetstore_metadata_service_read_file_metadata_duration_seconds",
+		Name:    "rafter_metadata_service_read_file_metadata_duration_seconds",
```
Comment (kazydek) [inline pkg/extractor/extractor.go]:
```suggestion
		Name:    "rafter_front_matter_service_read_file_metadata_duration_seconds",
```

File: pkg/processor/processor.go [hunk 4]
```diff
@@ -40,12 +40,12 @@ type Processor struct {
 
 var (
 	processSingleFileHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
-		Name:    "assetstore_metadata_service_process_single_file_duration_seconds",
+		Name:    "rafter_metadata_service_process_single_file_duration_seconds",
```
Comment (kazydek) [inline pkg/processor/processor.go]:
```suggestion
		Name:    "rafter_front_matter_service_process_single_file_duration_seconds",
```

File: pkg/processor/processor.go [hunk 5]
```diff
@@ -40,12 +40,12 @@ type Processor struct {
...
 		Help:    "Duration distribution of processing single file",
 		Buckets: prometheus.ExponentialBuckets(0.00001, 2, 16), // discuss about loading those values from envs, not only here but generally
 	})
 	processFilesHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
-		Name: "assetstore_metadata_service_process_all_files_duration_seconds",
+		Name: "rafter_metadata_service_process_all_files_duration_seconds",
```
Comment (kazydek) [inline pkg/processor/processor.go]:
```suggestion
		Name: "rafter_front_matter_service_process_all_files_duration_seconds",
```

**Comment by michal-hudy** [comment]
@kazydek good catch! I only checked assetstore/cms -> rafter....

**Pull request merged**